### ISSUE TITLE
[Suggestion] Disable Stable Physics

### ISSUE THREAD

**Issue by EterniaLogic** [godarklight/DarkMultiPlayer#341]
Disable physics to other players when a player is stable. i.e: a player is sitting on the ground, or a ship is not moving in orbit. This should prevent some obvious rubber-banding issues, so that players can dock easier.

**Comment by RockyTV** [comment]
Personally, I don't think this should be forced on the servers. Maybe a
setting could do it.

**Comment by EterniaLogic** [comment]
Yes, an option would be nice so that servers that are having this problem can just plop it on
